test/basic.html

```diff
@@ -128,6 +128,22 @@
     </template>
   </test-fixture>
 
+  <test-fixture id="FormWithPreventInvalidCustomElements">
+    <template>
+      <form is="iron-form" action="/responds_with_json" method="post">
+        <paper-input label="numbers" pattern="^[0-9]$" ></paper-input>
+      </form>
+    </template>
+  </test-fixture>
+
+  <test-fixture id="FormWithPreventInvalidElements">
+    <template>
+      <form is="iron-form" action="/responds_with_json" method="post">
+        <input type="text" name="letters" pattern="^[a-z]$" >
```

Comment:
`prevent-invalid` is a specific attribute on `iron-input`/`paper-input`, and i don't see it there. Maybe you just mean 'form with invalid pattern inputs' ?